frontend/src/static/js/store.js

```diff
@@ -8,6 +8,7 @@ const store = new Vuex.Store({
     fileTypeColors: {},
     loadingOverlayCount: 0,
     loadingOverlayMessage: '',
+    isTabActive: true,
```

Comment:
shouldn't this set to false by default?

Reply:
Thanks for pointing it out. Currently, `isTabActive` is set to true by default in the main window during its initialization. It is after a tab is selected and then closed that `isTabActive` will be set to false, so I set `isTabActive` to be true here in the store initially.